content/docs/pulumi-cloud/deployments/faq.md

```diff
@@ -21,6 +21,25 @@ aliases:
 
 Deployments run on single-use virtual machines and compute and storage are never shared across runs. We designed our architecture to maximize isolation. In addition, security features like OIDC allow you to fine tune credential scope, lifetime, and expiration policies at a per-deployment level. It is also possible to use [self-hosted runners](/docs/pulumi-cloud/deployments/customer-managed-agents/) if you require additional isolation.
 
+## Dependency Caching
+
+When using Pulumi-managed deployment agents, you have the option to speed up deployments using Dependency Caching.
```

Comment:
casing/emphasis: "Dependency Caching"->"*dependency caching*"

Instead of capitalizing the concept of "Dependency Caching" in the text, use it in lower case. However, the first time you introduce a new concept like this, you can use italics to indicate it is a special concept to understand (instead of proper noun casing).